openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -190,15 +190,14 @@ def test_save_related_send_emails(
     mock_send_webinar_emails = mocker.patch('openedx.adg.lms.webinars.admin.send_webinar_emails')
     mock_form_class = mocker.patch('openedx.adg.lms.webinars.forms.WebinarForm')
     mock_get_webinar_invitees_emails = mocker.patch('openedx.adg.lms.webinars.admin.get_webinar_invitees_emails')
-    mock_get_webinar_invitees_emails.return_value = []
+    mock_get_webinar_invitees_emails.return_value = ['test_invite@email.com']
```

Comment:
Let's keep it consistent and make constants for all the emails being used in the tests as we have made for the panelists and cohosts, also these emails are being repeated hence it would be a better approach to create constant for them.

Reply:
To keep it consistent, I hard coded this email like we already have it here
https://github.com/OmnipreneurshipAcademy/edx-platform/pull/509/files#diff-aff24b9f24fd56da6840ca0c88d9188cbac7748cb54fc56beefe2df3d682642fR211

Secondly IMO in test cases it would be better to create a constant only if we are using it in multiple test cases. But here scope is only a single test case. So we can hardcode these string for better readability specially in case of test cases.